src/services/__tests__/tourettes.test.js

```diff
@@ -22,7 +22,17 @@ describe('tourettes', () => {
       it('replies with at least three items from the profanities list', () => {
         const reply = sayTourettes(inputText);
         expect(reply).toNotBe(undefined);
-        expect(reply.split(' ').length).toBeGreaterThan(3);
+        expect(reply.split(' ').length).toBeGreaterThan(2);
```

Comment:
But for consistency, let's use greater than or equal to everywhere, don't you agree?